thespian/system/multiprocCommon.py

```diff
@@ -511,6 +510,28 @@ def shutdown_signal_detected(signum, frame):
         am.transport.interrupt_wait(signal_shutdown=True)
     return shutdown_signal_detected
 
+def get_min_log_level(logDefs:Dict[str, Any])->int:
+    """
+    Get the minimum logging level from a logging configuration definition. Tested here using example.
+    >>> import logging
+    >>> logDefs = {'version': 1, 'formatters': {'normal': {'format': '%(levelname)-8s %(message)s'}, 'actor': {'format': '%(levelname)-8s %(actorAddress)s => %(message)s'}}, 'filters': {'isActorLog': {'()': 'actorLogFilter'}, 'notActorLog': {'()': 'notActorLogFilter'}}, 'handlers': {'h1': {'class': 'logging.FileHandler', 'filename': 'example.log', 'formatter': 'normal', 'filters': ['notActorLog'], 'level': 20}, 'h2': {'class': 'logging.FileHandler', 'filename': 'example.log', 'formatter': 'actor', 'filters': ['isActorLog'], 'level': 20}}, 'loggers': {'': {'handlers': ['h1', 'h2'], 'level': 10}}}
```

Comment:
Multi-line input in doctest is supported with `...` prefixes for the subsequent lines, so this could be made a little more readable:

    >>> logDefs = {'version': 1,
    ...            'formatters': {
    ...              'normal': {
    ...           ... etc

Reply:
fair point, changes made